tools/CouchDB_Import_Demographics.php

```diff
@@ -188,6 +175,15 @@ function _generateQuery() {
             $fieldsInQuery .= $EDCFields;
             $groupBy .= ", c.EDC";
         }
+        // If consent is being used, add consent information into query
+        if ($config->getSetting("useConsent") === "true") {
+              $consentFields  = ", ct.Name AS Consent_type, COALESCE(cc.Status, 'not available') AS Consent_status, COALESCE(cc.DateWithdrawn, '0000-00-00') AS Consent_withdrawal";
```

Comment:
discuss "coalesce not available " ??

Reply:
when looking into using multiple consents in the DQT, i spoke to leigh mac about how IBIS does it and they coalesce "yes". I thought this was presumptuous so I opted for "not_available" instead, until the larger loris discussion of 'what should the default consent status be when one is not given?' can be had. I though about coalescing with "not_answered" instead (one of the enum options for consent status in the sql table) but I feel that this has a different connotation to "not_available".